grpc/v1/webserver.proto

```diff
@@ -113,17 +113,11 @@ message ListOptions {
   enum RepoListField {
     REPO_LIST_FIELD_UNKNOWN = 0;
     REPO_LIST_FIELD_REPOS = 1;
-    REPO_LIST_FIELD_MINIMAL = 2;
     REPO_LIST_FIELD_REPOS_MAP = 3;
   }
 
   // Field decides which field to populate in RepoList response.
   RepoListField field = 1;
-
-  // Return only Minimal data per repo that Sourcegraph frontend needs.
-  //
-  // Deprecated: use Field
-  bool minimal = 16;
```

Comment:
You should [reserve](https://protobuf.dev/programming-guides/proto3/#fieldreserved) the numbers of deleted protobuf fields to prevent future reuse.